### PR TITLE
Add special cases for TLCR

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: src/main/java/ca/on/oicr/gsi/migration/PinerySource.java [hunk 1]
```diff
@@ -884,7 +882,19 @@ public boolean modifyChild(MigrationHierarchyMember relative) {
     }
 
     /**
-     * Modifies the <pre>copyTo</pre> sample with information from the <pre>copyFrom</pre> sample. Intended for copying a field up and 
+     * Modifies the
```
Comment (callunity) [inline src/main/java/ca/on/oicr/gsi/migration/PinerySource.java]: weeeeird formatting

File: src/main/java/ca/on/oicr/gsi/migration/converters/project/TlcrSampleConverter.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  private static class CustomAliasConverter extends DefaultAttributeConversionVisitor {
+    @Override
+    public boolean convert(MigrationSample sample, String value) {
+      // try default conversion first before handling exceptions
+      if (SampleAttribute.NAME.convert(value, sample)) return true;
+      log.debug(
+          String.format("Handling special case for sample %s - %s = %s", sample.getSourceLimsId(), SampleAttribute.NAME.getKey(), value));
+      sample.setAlias(value);
```
Comment (callunity) [inline src/main/java/ca/on/oicr/gsi/migration/converters/project/TlcrSampleConverter.java]: For future PRs, could you provide examples of the names you will be handling with your converter?

Comment (apmasell) [inline src/main/java/ca/on/oicr/gsi/migration/converters/project/TlcrSampleConverter.java]: The TLCR project is sufficiently a disaster that I'll accept literally anything.

File: src/main/java/ca/on/oicr/gsi/migration/PinerySource.java (hunk 1 above)
Comment (apmasell) [inline src/main/java/ca/on/oicr/gsi/migration/PinerySource.java]: Will reformat the file when committing. :-/

**Pull request merged**